projects/gossip-service-typescript/src/service.ts

```diff
@@ -37,15 +37,30 @@ export default class GossipService {
   }
 
   askForHeartbeats() {
-    this.askForHeartbeat(this.peers.publicApi);
+    // this.askForHeartbeat(this.peers.publicApi);
```

Comment:
Why have you commented it out? Pinging other services won't work?

Reply:
It was a mistake. I've reversed it now.